tasks/kmt.py

```diff
@@ -669,3 +679,90 @@ def ssh_config(
                 print("    UserKnownHostsFile /dev/null")
                 print("    StrictHostKeyChecking accept-new")
                 print("")
+
+
+@task(
+    help={
+        "stack": "Name of the stack to get the status of. If None, show status of stack associated with current branch",
+        "all": "Show status of all stacks. --stack parameter will be ignored",
+    }
+)
+def status(ctx: Context, stack: Optional[str] = None, all=False):
```

Comment:
Can we add a `verbose` option to also print exactly which VMs/Instances are up/down?

Reply:
The current version should print the status of each VM/instance too without extra options, let me check

Reply:
<img width="681" alt="Screenshot 2024-03-22 at 14 25 53" src="https://github.com/DataDog/datadog-agent/assets/1334568/d73268f7-40db-434a-b258-cbac371f8164"> 

This is the output, I don't know if you were referring to something like this

Reply:
Yeah this is exactly what I meant